script/bootstrap.py

```diff
@@ -12,6 +12,7 @@
 SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
 VENDOR_DIR = os.path.join(SOURCE_ROOT, 'vendor')
 PYTHON_26_URL = 'https://chromium.googlesource.com/chromium/deps/python_26'
+CRACKPAD_URL = 'https://github.com/hokein/crashpad.git'
```

Comment:
I don't have admin right to do the movement. You may create a raw `crashpad` repo and add me as a collaborator.